File: example_mas/ChatDev/WareHouse/CLI_File_Renamer_Bulk_Pattern_CodeLlamaEval_20250729070719/main.java
import java.util.Scanner;
import java.io.File;
import java.io.IOException;
public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter the pattern: ");
        String pattern = scanner.nextLine();
        System.out.print("Enter the directory: ");
        String directory = scanner.nextLine();
        scanner.close();
        Renamer renamer = new Renamer(pattern, directory);
        renamer.renameFiles();
    }
}

File: example_mas/ChatDev/WareHouse/CLI_File_Renamer_Bulk_Pattern_CodeLlamaEval_20250729070719/main.py
import argparse
import os
from renamer import Renamer
def main():
    parser = argparse.ArgumentParser(description='CLI file renamer')
    parser.add_argument('-p', '--pattern', required=True, help='rename pattern')
    parser.add_argument('-d', '--directory', required=True, help='directory to rename files in')
    args = parser.parse_args()
    renamer = Renamer(args.pattern, args.directory)
    renamer.rename_files()
if __name__ == '__main__':
    main()